src/main/java/com/alipay/remoting/connection/AbstractConnectionFactory.java

```diff
@@ -178,6 +193,26 @@ private void initWriteBufferWaterMark() {
             lowWaterMark, highWaterMark));
     }
 
+    private SslContext initSSLContext() {
```

Comment:
The implementation of initSSLContext method is same as the one which is in the RpcServer.java class.I suggest you can add a class which called TlsUtils or TlsHelper and put this part of codes into this class.

Reply:
And for this initSSLContext method, I suggest you can add a parameter called forClient, so in this method you can add some part codes for client mode.

Reply:
Extract a method doesn't help the code more readable, but introduce if/else and more parameters. I think current implementation is acceptable and more readable.And if we have more SSL context stuffs  in future, we can refactor the code at that time.